app/services/triannon/ldp_loader.rb

```diff
@@ -99,7 +99,7 @@ def get_ttl sub_path = nil
     def statements_from_ttl_minus_fedora ttl
       # RDF::Turtle::Reader.new(ttl).statements.to_a
       g = RDF::Graph.new.from_ttl(ttl) if ttl
-      RDF::FCRepo4.remove_fedora_triples(g).statements if g
+      OA::Graph.remove_fedora_triples(g).statements if g
```

Comment:
Just curious about the meaning or specificity of `remove_fedora_triples` because generally triples are triples no matter where they are.  Also, just a little bit concerned about the 'truthy' evaluation for `if g`.  If the code gets to line 102, then `g` should be an RDF::Graph and it might be empty, which could be tested with `g.empty?` if that is what is meant or required here?  In other words, perhaps replace `if g` with `unless g.empty?` ?